src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-padding;
+  width: (7 * $cell-size) + (6 * $cell-gap);
```

Comment:
The width of the calendar is calculated directly in the SCSS file. It's a good practice to avoid such calculations in CSS files. Consider moving this logic to a JavaScript file. This is a code quality issue.